task/generate-olm-bundle/0.1/generate-olm-bundle.yaml

```diff
@@ -0,0 +1,40 @@
+---
+apiVersion: tekton.dev/v1beta1
+kind: Task
+metadata:
+  name: generate-olm-bundle
+spec:
+  description: Generate an OLM bundle
+  params:
+    - name: input-dir
+      description: Directory to read cluster-ready operator manifests from
+      default: deploy
+    - name: channels
+      description: Comma-separated list of channels the bundle belongs to
+      default: alpha
+    - name: version
+      description: Semantic version of the operator in the generated bundle
+    - name: package-name
+      description: Bundle's package name
+  workspaces:
+    - name: source
+      description: Workspace with the source code
+  steps:
+    - name: generate-olm-bundle
+      # TODO: ensure this image is built within konflux
+      # TODO: can this be configurable as a parameter? it should be specific to
+      # the target version
+      image: "registry.redhat.io/openshift4/ose-operator-sdk-rhel9:v4.16"
```

Comment:
Is it valid for this to be a parameter? This should be set to whatever version of OCP is being targetted

Reply:
I am not sure I understand. The bundle version is agnostic to the OCP version targeted. 

We prefer to not make the image a parameter as it makes the result of the task nondeterministic as people can substitute _and_ image in there.

Reply:
Sorry, I confused this one with the catalog task (https://redhat-internal.slack.com/archives/C04PZ7H0VA8/p1720615804304029?thread_ts=1720609182.337489&cid=C04PZ7H0VA8) which will have this problem

Reply:
We should remove this TODO then.